pkg/recordset/recordset_test.go

```diff
@@ -1,15 +1,776 @@
 package recordset
 
 import (
+	"io/ioutil"
 	"reflect"
+	"sort"
 	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/cloudformation"
 	"github.com/giantswarm/micrologger"
 )
 
```

Comment:
It would be good to align with https://github.com/giantswarm/fmt/blob/master/go/unit_tests.md. Especially for newly introduced tests.

Reply:
What should be aligned in your opinion ?

I see the `description` field which can be rename to `name` and add `case X` to them.
I skipped the error matcher here as I am not testing values return by functions but rather how they interact with AWS API. Any error which occur should result in a failure.

Reply:
> I see the description field which can be rename to name and add case X to them.

That.

And also fields should be specified explicitly in the struct initialization. I don't use linter myself but I can see other's IDEs explode because of that.

Reply:
Although I think we should deliver code conforming with our guidelines I find the other question more important. https://github.com/giantswarm/route53-manager/pull/11#discussion_r230727967